script/LaunchDeployer.s.sol

```diff
@@ -74,6 +74,13 @@ contract LaunchDeployer is FullDeployer {
         // Cache version hash to avoid redundant hash recalculation
         if (input.core.version == "3.1") _verifyAdmin(input.adminSafe);
 
+        // Pre-flight: if LayerZero should deploy, ensure PROTOCOL_ADMIN is set
+        if (input.adapters.layerZero.shouldDeploy) {
+            address protocolAdminEnv = vm.envAddress("PROTOCOL_ADMIN");
+            require(protocolAdminEnv != address(0), "PROTOCOL_ADMIN not set for LayerZero deployment");
+            console.log("LayerZero delegate (PROTOCOL_ADMIN):", vm.toString(protocolAdminEnv));
+        }
```

Comment:
NIT. We can consider `PROTOCOL_ADMIN` to always exist, otherwise, the `adminSafe` in `FullInput` will panic trying to fetch it